_episodes/03-skimage-images.md

```diff
@@ -361,13 +367,12 @@ plt.imshow(image, cmap="gray")
 {: .language-python}
 
 The first argument to `iio.imread()` is the filename of the image.
-The second argument `mode="L"` defines the type and depth of a pixel in the
-image (e.g., an 8-bit pixel has a range of 0-255). This argument is forwarded
-to the `pillow` backend, for which mode "L" means 8-bit pixels and
-single-channel (i.e., grayscale). `pillow` is a Python imaging library; which
-backend is used by `iio.imread()` may be specified (to use `pillow`, you would
-pass this argument: `plugin="pillow"`); if unspecified, `iio.imread()`
-determines the backend to use based on the image type.
+The second argument `mode="L"` defines the datatype and range of the pixel values in the image (e.g., an 8-bit pixel has a range of 0-255). This argument is forwarded to the `pillow` backend, a Python imaging library for which mode "L" means 8-bit pixels and single-channel (i.e., grayscale). The backend used by `iio.imread()` may be specified as an optional argument: to use `pillow`, you would
```

Comment:
```suggestion
The second argument `mode="L"` defines the type and range of the pixel values in the image (e.g., an 8-bit pixel has a range of 0-255). This argument is forwarded to the `pillow` backend, a Python imaging library for which mode "L" means 8-bit pixels and single-channel (i.e., grayscale). The backend used by `iio.imread()` may be specified as an optional argument: to use `pillow`, you would
```

* Line 371, you wrote just 'type' (and it's referring to the same thing, right?); 
* I'm not sure 'datatype' is a word yet; it's probably still 'data type' (remember when 'data set' became 'dataset');
* Maybe we could write 'data type' or 'data-type' (seeing both at https://numpy.org/doc/stable/reference/arrays.dtypes.html)?